src/components/sketches/test3.ts

```diff
@@ -0,0 +1,64 @@
+import p5 from 'p5';
+export default function sketch(p: p5): void {
+  let canvas: p5.Renderer;
+  let xo = 1;
```

Comment:
I dont think this sketch is ready to be published yet.. it seems a little unfinished (the rectangles and circles are not disappearing after a while

Reply:
yeah let's stash it!